src/docfx/lib/markdown/MarkdownEngine.cs

```diff
@@ -144,6 +145,30 @@ public string ToHtml(MarkdownObject markdownObject)
             return sb.ToString();
         }
 
+        // patch of CodeInlineRenderer, will be fixed in upstream. todo
```

Comment:
consider put this class after all methods

Reply:
ok

Reply:
https://github.com/lunet-io/markdig/issues/495